syntax = "proto3";

// Returns details of ZenPlayer states and accepts ZenPlayer commands
service ZenPlayerService {
    rpc get_details(TrackDetailsRequest) returns (Result);
}

// Get the details of the given track
message TrackDetailsRequest {
    string artist = 1;
    string album = 2;
    string fileName = 3;
}

// Return details of the requested track
message Result {
    string text = 1;
}
